wpilibc/sim/src/RobotDrive.cpp

```diff
@@ -20,7 +20,8 @@ using namespace frc;
 
 const int RobotDrive::kMaxNumberOfMotors;
 
-static auto make_shared_nodelete(SpeedController* ptr) {
+static std::shared_ptr<SpeedController> make_shared_nodelete(
```

Comment:
This should have been `decltype(auto)`, which would be deduced as `std::shared_ptr<SpeedController>&&`.